catalog/core/catalog-core-commons/src/main/java/ddf/util/XPathHelper.java

```diff
@@ -250,14 +258,37 @@ public synchronized Object evaluate(String xpathExpressionKey, QName returnType,
                 .setNamespaceContext(nsContext);
 
         XPathExpression compiledExpression = XPathCache.getCompiledExpression(xpathExpressionKey);
-
         Thread thread = Thread.currentThread();
         ClassLoader loader = thread.getContextClassLoader();
         thread.setContextClassLoader(this.getClass()
                 .getClassLoader());
 
+        DocumentBuilderFactory df =DocumentBuilderFactory.newInstance();
+        df.setNamespaceAware(true);
         try {
-            return compiledExpression.evaluate(document, returnType);
+            df.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
+            df.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
+        } catch (ParserConfigurationException e) {
+            throw new XPathExpressionException(e);
+        }
+        DocumentBuilder documentBuilder;
+        byte[] array;
+        try (ByteArrayOutputStream bos = new ByteArrayOutputStream()) {
+            documentBuilder = df.newDocumentBuilder();
+            TransformerFactory transformerFactory = TransformerFactory.newInstance();
+            Transformer transformer = transformerFactory.newTransformer();
+            DOMSource source = new DOMSource(document);
+            StreamResult result = new StreamResult(bos);
+            transformer.transform(source, result);
+            array = bos.toByteArray();
+        } catch (IOException | ParserConfigurationException | TransformerException e) {
+            throw new XPathExpressionException(e);
```

Comment:
The other catch block in this method is doing the same thing. I'm just being consistent.